docs/developer/file-structure.rst

```diff
@@ -10,8 +10,8 @@ The PyMedPhys repository has the following general structure:
 
    pymedphys/
    |   README.rst
-   |   LICENSE
-   |   Apache-2.0
+   |   LICENSE-AGPL-3.0-or-later
+   |   LICENSE-Apache-2.0
    |   changelog.md
    |   setup.py
    |   layers.yml
```

Comment:
We can leave it there for now, but after the big split we are no longer running layer linter. So until that gets sorted should we just leave the layer description out?